olive/passes/onnx/tensorrt/trt_dla_transforms.py

```diff
@@ -0,0 +1,106 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+from pathlib import Path
+from typing import Dict, Type
+
+import onnxscript
+from onnxscript.rewriter import pattern
+
+from olive.hardware.accelerator import AcceleratorSpec
+from olive.model import ONNXModelHandler
+from olive.model.utils import resolve_onnx_path
+from olive.passes import Pass
+from olive.passes.onnx.common import get_external_data_config, model_proto_to_olive_model
+from olive.passes.pass_config import BasePassConfig, PassConfigParam
+
+logger = logging.getLogger(__name__)
+
+
+class TrtMatMulToConvTransform(Pass):
+    """Convert 2Dx2D and 3Dx2D MatMul to Transpose-Conv-Transpose to meet HW restriction."""
+
+    @classmethod
+    def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
+        return get_external_data_config()
+
+    def _run_for_config(
+        self, model: ONNXModelHandler, config: Type[BasePassConfig], output_model_path: str
+    ) -> ONNXModelHandler:
+        output_model_path = resolve_onnx_path(output_model_path, Path(model.model_path).name)
+        model_proto = model.load_model()
```

Comment:
```suggestion
        model = ir.load(model.model_path)
```

Use ir.load() to load the model

Reply:
You may now use `model.load_ir_model()` from https://github.com/microsoft/Olive/pull/1771